doc/texts.md

```diff
@@ -2,39 +2,9 @@
 
```

Comment:
Unrelated

Reply:
I had already changed this in my PR, but I'll revert my changes